Assets/MixedRealityToolkit/Definitions/Utilities/ProfileMenuItemIndices.cs

```diff
@@ -29,7 +29,7 @@ public enum CreateProfileMenuItemIndices
         EyeTracking,
         MouseInput,
         SceneSystem,
-
+        SceneUnderstandingObserver,
```

Comment:
nit: keep the blank line, to make it more obvious that `Assembly` should stay at the end